brendenwest/cascade
# Implementation of the UI library React-Native-Elements

## Description
Closes #11 

React-Native-Elements was installed according to their documentation (https://reactnativeelements.com/docs/), including the addition of react-native-vector-icons and react-native-safe-area-context package. I added icons to the existing buttons for the illustration purposes
![Screenshot 2021-10-16 144852](https://user-images.githubusercontent.com/17448320/137602908-98f65ffb-40ff-4e6c-a8d0-aadf98221f25.png)

## To Test
1. Switch to ab-11-nativeelements branch
2. Run npm install
2. Start the app with npx react-native run-android
3. Check if everything is loaded and works correctly
